dataverse-webapp/src/main/webapp/dashboard-licenses.xhtml

```diff
@@ -94,7 +94,7 @@
                     </p:tab>
 
                     <p:tab title="#{bundle['dashboard.otherTermsOfUse.header']}">
-                        <p:dataTable var="otherTos" class="table-othertos" value="#{OtherTermsOfUseTab.otherTermsOfUseDto}">
+                        <p:dataTable var="otherTos" class="table-othertos no-min-width" value="#{OtherTermsOfUseTab.otherTermsOfUseDto}">
```

Comment:
From the task description:
`The last column should be labeled "Active" (NOT "Active by Default") - same thing with the column name on "Licenses" tab` - this is not addressed in this pr

Reply:
Oh I haven't noticed there is something under the picture